studio/pages/project/[ref]/functions/[functionSlug]/index.tsx

```diff
@@ -135,28 +222,140 @@ const PageLayout: NextPageWithLayout = () => {
               )
             }}
           />
-          <ReportWidget
-            title="Invocations"
-            data={chartData}
-            isLoading={isChartLoading}
-            renderer={(props) => (
-              <BarChart
-                className="w-full"
-                xAxisKey="timestamp"
-                yAxisKey="count"
-                data={props.data}
-                highlightedValue={sumBy(props.data, 'count')}
-                customDateFormat={selectedInterval.format}
-                onBarClick={(v) => {
-                  router.push(
-                    `/project/${projectRef}/functions/${functionSlug}/invocations?its=${startDate.toISOString()}&ite=${
-                      v.timestamp
-                    }`
+          {resourceUsageMetricsEnabled ? (
+            <ReportWidget
+              title="Invocations"
+              data={invocationsChartData}
+              isLoading={isReqStatsChartLoading}
+              renderer={(props) => {
+                const data = props.data
+                  .map((d: any) => [
+                    {
+                      status: '2xx',
+                      count: d.success_count,
+                      timestamp: isUnixMicro(d.timestamp)
```

Comment:
if already normalized above, shouldn't need to do this.

Reply:
removed